### PR TITLE
Document Windows limitation to 4 controllers in Controllers, gamepads and joysticks

### PR DESCRIPTION
**Pull request by Calinou**
- This closes https://github.com/godotengine/godot/issues/85937.

### CODE REVIEW

File: tutorials/inputs/controllers_gamepads_joysticks.rst [hunk 1]
```diff
@@ -293,6 +293,10 @@ First, check that your controller is recognized by other applications. You can
 use the `Gamepad Tester <https://gamepad-tester.com/>`__ website to confirm that
 your controller is recognized.
 
+Also, on Windows, Godot only supports up to 4 controllers at a time. This is
```
Comment (skyace65) [inline tutorials/inputs/controllers_gamepads_joysticks.rst:296]:
```suggestion
On Windows Godot only supports up to 4 controllers at a time. This is
```
I think this reads a bit better, but I'd also be fine with merging as is.

**Review by skyace65 — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]
Agree with skyace65, besides that this LGTM

**Comment by zolrath** [comment]
To clarify, when using controllers like Switch Pro controllers that don't use XInput, I can use 4 xbox controllers + additional switch pro controllers. As I only have 3, I've tested up to 4 xbox controllers + 3 switch pro controllers + keyboard to test my game.

**Comment by mhilbrunner** [comment]
Merged. Thanks for contributing! 🎉

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.2.

**Pull request merged**